cmd/lncli/commands.go

```diff
@@ -1709,6 +1709,12 @@ var addInvoiceCommand = cli.Command{
 				"specified an expiry of 3600 seconds (1 hour) " +
 				"is implied.",
 		},
+		cli.BoolTFlag{
```

Comment:
TIL about `BoolTFlag`! 😀

Reply:
I don't think we can quite flip this switch yet. We'll need to wait until we know _all_ the other implementations support height hints, and also our implementation. As a result, this should be changed to a regular `Bool` flag, with the default being `false`.

Reply:
bool af